xdsl/ir/core.py

```diff
@@ -388,6 +389,11 @@ def __eq__(self, other: object) -> bool:
         return self is other
 
 
+Operand: TypeAlias = SSAValue
+VarOperand: TypeAlias = list[Operand]
```

Comment:
oh I forgot this was a list, that's weird

Reply:
Indeed, let's think about it independently though if that's fine to you? Apparently, it ain't broken